lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data.fetch(:from)[:name]
```

Comment:
Just adding on to what @claudiob was saying. The other methods return nil if it can't find the attribute in the data hash.

For example:
```ruby
    def description
      data[:description]
    end
```
If for some reason, the `:description` is not there, it returns nil, otherwise it returns the String of the description. We actually don't have any tests to see if any of those attributes return nil if those attributes did not existed. We have been assuming that they always exists.

Maybe to keep consistent, we could have `page_name` return `nil` if it can't access `:from`? Something like `data.fetch(:from, {})[:name]`, even though according to the tests, we generally assume that those attributes should be there anyways.

@claudiob @kangkyu ??